README.md

```diff
@@ -47,3 +47,7 @@ series-renamers --directory [path/to/target/directory] --prefix 'Series S01 E' [
 | `--directory`, `-d` | `string`  | path to target directory | `false`  | current directory |
 | `--prefix`, `-p`    | `string`  | new file name prefix     | `true`   |                   |
 | `--dry-run`, `-dr`  | `boolean` | only preview the results | `false`  | `false`           |
+
+## License
+
+Series Renamer Tools is an open-source tool licensed under [MIT License](https://github.com/kartikeysingh6/series-renamer-tools/blob/master/LICENSE).
```

Comment:
I think you should remove https://github.com/kartikeysingh6/series-renamer-tools/blob/master/

```suggestion
Series Renamer Tools is an open-source tool licensed under [MIT License](LICENSE).
```